src/sentry/static/sentry/app/views/settings/components/forms/radioField.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import styled from 'react-emotion';
+import PropTypes from 'prop-types';
+
+import InputField from './inputField';
+import {growIn} from './styled/animations';
+
+class RadioField extends InputField {
+  static propTypes = {
+    id: PropTypes.number,
+    value: PropTypes.string,
+    choices: PropTypes.func,
+  };
+
+  onChange = (value, onChange, onBlur, e) => {
+    onChange(value, e);
+    onBlur(value, e);
+  };
+
+  isSelected = ({value, id}) => {
+    return value ? value === id : id === 0;
+  };
+
+  render() {
+    return (
+      <InputField
+        {...this.props}
+        field={({onChange, onBlur, value, disabled, ...props}) => (
+          <div role="radiogroup" aria-labelledby={props.label}>
```

Comment:
Maybe turn this into a `<RadioGroup>` component that takes props. Then I think the tests you were trying to write will become easier since you can test `<RadioGroup>` vs the field that's wrapped with `FormField`